libs/CCAppCommon/include/ccApplicationBase.h

```diff
@@ -24,15 +24,26 @@
 //! Mimic Qt's qApp for easy access to the application instance
 #define ccApp (static_cast<ccApplicationBase *>( QCoreApplication::instance() ))
 
+
+/*
+ * main ccApplicationBase that the CCViewer which implements
```

Comment:
So I think that's a general comment: I dont' see the point of describing the inheritance (which can be very easily visualised in doxygen) and copying / referencing Qt's documentation explicitly (and pointing to version 5, it will be hard to maintain in the future)